migrateTo_7_1_1/src/test/groovy/org/bonitasoft/migration/CheckMigratedTo7_1_1.groovy

```diff
@@ -0,0 +1,141 @@
+                .done()).getResult().get(0))
+                .isEqualToIgnoringGivenFields(new FormMapping(processDefinitionId: processDefinitionId, type: TASK, task: "step1", target: INTERNAL), "id", "pageMappingKey", "lastUpdateDate")
+        assertThat(processAPI.searchFormMappings(new SearchOptionsBuilder(0, 10)
+                .filter(FormMappingSearchDescriptor.TYPE, TASK).filter(FormMappingSearchDescriptor.TASK, "step2").filter(PROCESS_DEFINITION_ID, processDefinitionId)
+                .done()).getResult().get(0))
+                .isEqualToIgnoringGivenFields(new FormMapping(processDefinitionId: processDefinitionId, type: TASK, task: "step2", target: INTERNAL, pageId: page.getId()), "id", "pageMappingKey", "lastUpdateDate")
+        assertThat(processAPI.searchFormMappings(new SearchOptionsBuilder(0, 10)
+                .filter(FormMappingSearchDescriptor.TYPE, TASK).filter(FormMappingSearchDescriptor.TASK, "step3").filter(PROCESS_DEFINITION_ID, processDefinitionId)
+                .done()).getResult().get(0))
+                .isEqualToIgnoringGivenFields(new FormMapping(processDefinitionId: processDefinitionId, type: TASK, task: "step3", target: LEGACY), "id", "pageMappingKey", "lastUpdateDate")
+        assertThat(processAPI.searchFormMappings(new SearchOptionsBuilder(0, 10)
+                .filter(FormMappingSearchDescriptor.TYPE, TASK).filter(FormMappingSearchDescriptor.TASK, "step4").filter(PROCESS_DEFINITION_ID, processDefinitionId)
+                .done()).getResult().get(0))
+                .isEqualToIgnoringGivenFields(new FormMapping(processDefinitionId: processDefinitionId, type: TASK, task: "step4", target: FormMappingTarget.URL), "id", "pageMappingKey", "lastUpdateDate")
+        assertThat(processAPI.searchFormMappings(new SearchOptionsBuilder(0, 10)
+                .filter(FormMappingSearchDescriptor.TYPE, TASK).filter(FormMappingSearchDescriptor.TASK, "step5").filter(PROCESS_DEFINITION_ID, processDefinitionId)
+                .done()).getResult().get(0))
+                .isEqualToIgnoringGivenFields(new FormMapping(processDefinitionId: processDefinitionId, type: TASK, task: "step5", target: FormMappingTarget.NONE), "id", "pageMappingKey", "lastUpdateDate")
+
+
+    }
+
+    @Test
+    def void check_quartz_table_is_migrated() {
+
+        def dburl = System.getProperty("dburl");
+        def dbuser = System.getProperty("dbuser")
+        def dbpassword = System.getProperty("dbpassword")
+        def dbdriverClass = System.getProperty("dbdriverClass")
+
+        def db = [url: dburl, user: dbuser, password: dbpassword, driver: dbdriverClass]
+
+        def sql = Sql.newInstance(db.url, db.user, db.password, db.driver)
+
+        // will fail with error:
+        // org.postgresql.util.PSQLException: ERROR: column "long_prop_1" does not exist
+        // if migration is not done as expected
+
+        sql.execute('''
```

Comment:
since migration step depends on db vendor and initial platform installation version, i prefer to keep it as a integration test